src/main/java/com/project/service/UserServiceImpl.java

```diff
@@ -46,24 +45,24 @@ public User addUser(User user) throws UserException {
         user.setId(null);//do not allow choosing id
 
         user.setPassword(bcryptEncoder.encode(user.getPassword()));
-        if(userRepository.findByEmail(user.getEmail()).isPresent())
+        if (userRepository.findByEmail(user.getEmail()).isPresent())
             throw new UserEmailExistsException(user.getEmail());
 
         List<String> roles = new ArrayList<>();
-        roles.add("User");
+        roles.add("USER");
         user.setRoles(roles); //probably more logical as we don't have admin registration
 
         return userRepository.save(user);
     }
 
     @Override
     public User updateUser(User user, long id) throws UserException {
-        if(!userRepository.findById(id).isPresent())
+        if (!userRepository.findById(id).isPresent())
             throw new UserNotFoundException(id);
 
         userValidator.validate(user);
 
-        if(userRepository.findByEmail(user.getEmail()).isPresent()
+        if (userRepository.findByEmail(user.getEmail()).isPresent()
```

Comment:
`if` statements should be opened and closed with curly braces `{}` even if they have only one line of code.

Reply:
added curly braces {} for if statements in UserServiceImpl and ApplicationServiceImpl classes